### PR TITLE
fix(docs): documentation, API, and blog typos

### PR DESCRIPTION
**Pull request by imbagila**
This PR is for fixing the typos from the documentation, API, and the blog.

### CODE REVIEW

File: apps/docs/docs/learn/02-thinking-in-stylex.mdx [hunk 1]
```diff
@@ -303,7 +303,7 @@ the styles applied directly on an element always take precedence over inherited
 styles.
 
 This is often not the case when using complex selectors as the complex selectors
-usually have higher specificity than the simple classname selectors used for
+usually have higher specificity than the simple `className` selectors used for
```
Comment (necolas) [inline apps/docs/docs/learn/02-thinking-in-stylex.mdx:306]:
```suggestion
usually have higher specificity than the simple class selectors used for
```

**Review by necolas — COMMENTED** [review]
Thanks. Some of these overlap with fixes in another PR; you'll need to rebase once we merge that one.

**Review by nmn — CHANGES_REQUESTED** [review]
Requesting the change that this PR be rebased!

**Comment by necolas** [comment]
I rebased and merged

**Pull request closed without merging**